docs/unified-test-documentation/dasharo-compatibility/326-DTS-firmware-update.md

```diff
@@ -12,7 +12,53 @@
 1. Make yourself familiar with
     [Bootable over network](../../../common-coreboot-docs/dasharo_tools_suite/#bootable-over-network)
 
-### FDT001.001 Firmware update by using Dasharo Tools Suite
+### FDT001.001 Flash firmware by using Dasharo Tools Suite
+
+**Test description**
+
+This test aims to verify whether there is the possibility to flash the DUT
+firmware by using Dasharo Tools Suite (DTS).
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+1. [Disable Secure Boot](../../unified-test-documentation/dasharo-security/206-secure-boot.md).
+1. Obtain `coreboot.rom` binary.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Press `BOOT_MENU_KEY` to enter the boot menu.
+1. In the `Boot Menu`, select the `iPXE Network Boot` option.
+1. In the `Network Boot menu`, select the `Dasharo Tools Suite` option.
```

Comment:
This is not a universal solution. Some devices have hardcoded Dasharo Tools Suite in the ipxe menu, some do not have such an option, so we should use ipxe shell and need a net interface to be configured.

Reply:
appiled